drake/systems/LQR.h

```diff
@@ -0,0 +1,52 @@
+#ifndef DRAKE_LQR_H
+#define DRAKE_LQR_H
+
+#include "Core.h"
+#include "LinearSystem.h"
+#include "drakeGradientUtil.h"
+#include "drakeUtil.h"
+
+namespace Drake {
+
+  template <typename Derived, template<typename> class StateVector, template<typename> class InputVector, template<typename> class OutputVector, bool isTimeVarying, bool isDirectFeedthrough>
+  std::shared_ptr<AffineSystem<UnusedVector,StateVector,InputVector> > timeInvariantLQR(const System<Derived,StateVector,InputVector,OutputVector,isTimeVarying,isDirectFeedthrough>& sys, const StateVector<double>& x0, const InputVector<double>& u0, const Eigen::MatrixXd& Q, const Eigen::MatrixXd& R) {
+    const unsigned int num_states = StateVector<double>::RowsAtCompileTime;
+    const unsigned int num_inputs = InputVector<double>::RowsAtCompileTime;
```

Comment:
Shouldn't be unsigned. Also need to handle case where `StateVector<double>::RowsAtCompileTime != x0.size()`, same for inputs.